pavement.py

```diff
@@ -237,12 +241,12 @@ def generate_locale():
 
     path("includes.txt").remove()
 
-    print "Locale generated"
+    print("Locale generated")
 
 
 @task
 def tests():
-    call(["nosetests2"])
+    call(["nosetests"])
```

Comment:
Codacy found an issue: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=4846282)